api/integrations/lead_tracking/hubspot/client.py

```diff
@@ -64,8 +67,11 @@ def create_contact(self, user: FFAdminUser, hubspot_company_id: str) -> dict:
         )
         return response.to_dict()
 
-    def create_company(self, name: str) -> dict:
-        properties = {"name": name}
+    def create_company(self, name: str, active_subscription: str) -> dict:
+        properties = {
+            "name": name,
+            "active_subscription": active_subscription,
```

Comment:
same comment as on the other PR, I've love to pull this out as a constant.

Reply:
As discussed on the other PR we decided against it.